torch/_export/verifier.py

```diff
@@ -31,6 +31,35 @@ def _check_has_fake_tensor(node: torch.fx.Node) -> None:
     return _check_val(node)
 
 
+def check_nn_module_stack(graph_module: torch.fx.GraphModule) -> None:
```

Comment:
Could we also call this function directly inside verifier?

Reply:
That was the original idea/implementation, but re. the comments in test/export/test_export_nn_module_stack.py, this check still fails with tests in serialization/export passes :(

I was hoping to add that file to add verification onto tests in test_export.py for now, but agree that this isn't very elegant.